tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Documentation  Test 9-2 - VICAdmin CertAuth
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Keywords ***
+Curl
+    [Arguments]  ${path}
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}${path}
```

Comment:
OH YOURE RIGHT
I will do that
I actually just learned about `%{env_var}` after I wrote that hardcoded path :+1: